web/components/input/buttons/index.js

```diff
@@ -0,0 +1,7 @@
+import PrimaryButton from './PrimaryButton';
```

Comment:
Question about the location of the button component: is button an input?

Reply:
Absolutely! The [standard way to declare buttons used to be `<input type="button" />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/button)